src/App.tsx

```diff
@@ -1,61 +1,112 @@
 import './App.scss';
+import { useState } from 'react';
+import { TodoList } from './components/TodoList';
 
```

Comment:
Consider using destructuring to import `useState` from `react`. It will make the code more concise and readable. 🤓